app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
```

Comment:
better to use `()` then `\`

Reply:
Can you explain? I'm trying to move the code to the next line, because when I check with pep8 I get an error about the line length being exceeded. The IDE itself substituted "/" during the transfer.

Reply:
It is a good practice to avoid line-breaking operator `\` when defining a method's signature. Prefer placing each argument including `self` on a new line so there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this change here and in the `__init__` for `CarWashStation` class as well